packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -126,6 +116,30 @@ export const ContentPanel: FC = () => {
         [runSqlQuery, sql, limit],
     );
 
+    const activeConfigs = useAppSelector((state) => {
+        const configsWithTable = state.sqlRunner.activeConfigs
+            .map((type) => selectChartConfigByKind(state, type))
+            .filter(
+                (config): config is NonNullable<typeof config> =>
+                    config !== undefined,
+            );
+
+        const tableConfig = configsWithTable.find(isTableChartSQLConfig);
+        const chartConfigs = configsWithTable.filter(
+            (
+                c,
+            ): c is Exclude<
+                NonNullable<ReturnType<typeof selectChartConfigByKind>>,
+                TableChartSqlConfig
+            > => !isTableChartSQLConfig(c),
+        );
+
+        return {
+            chartConfigs,
+            tableConfig,
+        };
+    });
```

Comment:
I could make this into its own selector separately, but would have to be a separate PR